pokedex/pokemon.lua

```diff
@@ -456,8 +457,8 @@ function M.get_defaut_max_hp(pkmn)
 				at_level = table.remove(evolutions)
 				local _, from_level = next(evolutions)
 				from_level = from_level or M.get_caught_level(pkmn)
-				local hit_dice = pokedex.get_pokemon_hit_dice(from_pkmn)
-				local hit_dice_current = pokedex.get_pokemon_hit_dice(current)
+				local hit_dice = pokedex.get_hit_dice(from_pkmn)
```

Comment:
This appears to just have been broken?

Reply:
Must have broke after the refactor were the pokedex.lua / pokemon.lua became a bit clearer (so not released yet)